assets/containerisableComponents/marketingConsent/marketingConsent.jsx

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+// ----- Imports ----- //
+
+import * as React from 'react';
+import CtaLink from 'components/ctaLink/ctaLink';
+import CheckboxInput from 'components/checkboxInput/checkboxInput';
+import ErrorMessage from 'components/errorMessage/errorMessage';
+import DotcomCta from 'components/dotcomCta/dotcomCta';
+import PageSection from 'components/pageSection/pageSection';
+
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  consentApiError: boolean,
+  confirmOptIn: ?boolean,
+  onClick: (marketingPreferencesOptIn: boolean, email?: ?string, csfr: CsrfState) => void,
```

Comment:
`csfr` -> `csrf`

Reply:
Actually, I can delete all the names. It is a type 😄